xwiki-xwikitest-script/xwikitest

```diff
@@ -500,7 +500,7 @@ while getopts ":hb:B:d:e:f:i:I:kl:L:np:r:s:t:u:U:v:wz" OPT; do
         u)  # Save the data folder
             XWDATABKP=$OPTARG
             ;;
-        U)  # Save the data folder (not doing anything elsei but export)
+        U)  # Save the data folder (not doing anything else but export)
```

Comment:
Thanks, but note that this does not count towards your CGI task, since this is a code comment in a code file and not a translation in a translations file. Remember to focus on the 2 file names ("ApplicationResources.properties" or "Translations.xml").

Reply:
oh nooooo. :((((

Reply:
I did review almost all of them but I can't find any more errors.After the second page, French or other language code starts and there is no English code .